profiles/rocket_profile.py

```diff
@@ -37,7 +37,7 @@ def expected_devices(self) -> List[DeviceType]:
 
     @property
     @abstractmethod
-    def mapping_device(self) -> DeviceType:
+    def mapping_device(self) -> List[DeviceType]:
```

Comment:
```suggestion
    def mapping_devices(self) -> List[DeviceType]:
```
Plural might make this more clear now. I noticed you already changed this in other places 👍